rpy2/situation.py

```diff
@@ -13,6 +13,11 @@
 from typing import Optional
 import warnings
 
+if sys.maxsize > 2**32
```

Comment:
I think you missed a colon:
```suggestion
if sys.maxsize > 2**32:
```